scripts/core/registry.yml

```diff
@@ -364,3 +364,9 @@ etors:
 - name: USM_POOL_GET_INFO
   desc: Enumerator for $xUSMPoolGetInfo
   value: '120'
+- name: USM_IMPORT_EXP
+  desc: Enumerator for $xUSMImportExp
+  value: '122'
+- name: USM_RELEASE_EXP
+  desc: Enumerator for $xUSMReleaseExp
+  value: '123'
```

Comment:
When you rebase - I recommend deleting these additions. After you rebase you can then rerun the generate script and it will assign you new values for these entry points. This is an unfortunate consequence of how the registry is generated.